# Represents a KDL::RigidBodyInertia instance.
#
# 6D Inertia of a rigid body
#

float64 m
Vector cog
RotationalInertia I
